package.json

```diff
@@ -26,6 +26,7 @@
     "genomic-reader": "^1.4.7",
     "genomics-formats": "github:VALIS-software/Genomics-Formats",
     "mkdirp": "^0.5.1",
+    "parcel": "^1.12.3",
```

Comment:
@hitz Can this be changed to "1.12.3" from "^1.12.3"; to ensure minor and patch parts are not updated in the future?